From abc50146600eb2cb93aec321d003970296950343 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Mon, 13 May 2024 16:42:56 +0100
Subject: [PATCH 1086/1135] staging: bcm2835-codec: 32bpp RGB formats need a
 64byte alignment

The firmware needs 16 pixel alignment on RGBx 32bpp formats, which
would be 64 byte. The driver was only setting 32byte alignment.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 .../staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c  | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
+++ b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
@@ -256,14 +256,14 @@ static const struct bcm2835_codec_fmt su
 	}, {
 		.fourcc			= V4L2_PIX_FMT_BGR32,
 		.depth			= 32,
-		.bytesperline_align	= { 32, 32, 32, 32, 32 },
+		.bytesperline_align	= { 64, 64, 64, 64, 64 },
 		.flags			= 0,
 		.mmal_fmt		= MMAL_ENCODING_BGRA,
 		.size_multiplier_x2	= 2,
 	}, {
 		.fourcc			= V4L2_PIX_FMT_RGBA32,
 		.depth			= 32,
-		.bytesperline_align	= { 32, 32, 32, 32, 32 },
+		.bytesperline_align	= { 64, 64, 64, 64, 64 },
 		.flags			= 0,
 		.mmal_fmt		= MMAL_ENCODING_RGBA,
 		.size_multiplier_x2	= 2,
